packages/react-components/react-tags-preview/src/contexts/useAvatarContextValues.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { AvatarSize, AvatarShape } from '@fluentui/react-avatar';
+
+export type UseAvatarContextValuesOptions = {
+  avatarSize: AvatarSize;
+  avatarShape: AvatarShape;
+};
+
+export type AvatarContextValues = {
+  avatar: {
+    size?: AvatarSize;
+    shape?: AvatarShape;
+  };
+};
+
+export function useAvatarContextValues_unstable(state: UseAvatarContextValuesOptions): AvatarContextValues {
```

Comment:
This signature might be a little weird to be exported from `react-tags` package?! perhaps having a name more associated to tags would be better?!

Reply:
renamed to useTagAvatarContextValues_unstable :)